mpeg2dec.c

```diff
@@ -1635,7 +1635,7 @@ int stream_component_open(VideoState *is, int stream_index)
     AVCodecContext *codecCtx;
     AVCodec *codec;
     AVCodec *codec_hw = NULL;
-
+	char strResize[11];

```

Comment:
Can you increase the string size for safety?

Reply:
Sure. Was thinking #####x#####, so 5 digits for each. But I can update, no issue. Suggestions?